tests/test_drafts.py

```diff
@@ -65,3 +65,16 @@ def test_delete_draft(api_client):
     draft.save()
     # ... and delete it for real.
     draft.delete()
+
+
+@responses.activate
+@pytest.mark.usefixtures("mock_draft_saved_response")
+def test_draft_version(api_client):
+    draft = api_client.drafts.create()
+    assert 'version' not in draft
```

Comment:
The [`create()`](https://github.com/singingwolfboy/nylas-python/blob/master/nylas/client/restful_models.py#L26-L50) method will populate the object based on the `kwargs` passed into this function. So to create an object with an initial version of 0, you can do:

```python
draft = api_client.drafts.create(version=0)
```

However, there's no special handling for draft numbers in the `create()` method itself, so if you leave it unspecified, then it won't be in resulting Draft instance. Do you want me to change that, so that an unsaved Draft instance automatically has a version number of 0?

The draft numbers in this particular test are coming from the `mock_draft_saved_response` fixture. Right now, the fixture will increase the draft number by one every time. Do you want me to change the fixture so that the first save returns a version number of 0, instead of 1?

Note that if you want both of these modifications, that means that the first time you save a Draft instance, the version number _will not change_. Is that correct?